Cannot build external-dns because of failed external dependencies

I'm on a pristine checkout (623ecd07f00cdda70953924fc7d2ee58720d1231) of master and run:

`make build`

Its output is:
`
CGO_ENABLED=0 go build -o build/external-dns -v -ldflags "-X github.com/kubernetes-incubator/external-dns/pkg/apis/externaldns.Version=v0.5.11-11-g623ecd07 -w -s" .
provider/azure.go:28:2: cannot find package "github.com/Azure/azure-sdk-for-go/arm/dns" in any of:
        /usr/lib/go-1.10/src/github.com/Azure/azure-sdk-for-go/arm/dns (from $GOROOT)
        /home/dirk/go/src/github.com/Azure/azure-sdk-for-go/arm/dns (from $GOPATH)
provider/digital_ocean.go:28:2: cannot find package "github.com/digitalocean/godo/context" in any of:
        /usr/lib/go-1.10/src/github.com/digitalocean/godo/context (from $GOROOT)
        /home/dirk/go/src/github.com/digitalocean/godo/context (from $GOPATH)
make: *** [Makefile:57: build/external-dns] Error 1
`

Have those external dependencies been moved around?

--- a/docs/contributing/getting-started.md
+++ b/docs/contributing/getting-started.md
@@ -2,7 +2,7 @@
 
 ### Building
 
-You can build ExternalDNS for your platform with `make build`, you may have to install the necessary dependencies with `sudo make dep`. The binary will land at `build/external-dns`.
+You can build ExternalDNS for your platform with `make build`, you may have to install the necessary dependencies with `make dep`. The binary will land at `build/external-dns`.
 
 ### Design
 

--- a/docs/contributing/getting-started.md
+++ b/docs/contributing/getting-started.md
@@ -2,7 +2,7 @@
 
 ### Building
 
-You can build ExternalDNS for your platform with `make build`. The binary will land at `build/external-dns`.
+You can build ExternalDNS for your platform with `make build`, you may have to install the necessary dependencies with `sudo make dep`. The binary will land at `build/external-dns`.
 
 ### Design
 

